Tasks/DropValidatorV0/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dropvalidator",
+  "version": "0.1.0",
+  "description": "Validates a given manifest with a manifest file to verify its authenticity.",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
```

Comment:
I don't think this is the canonical address of the repo anymore

Reply:
Fixed